### PR TITLE
Create multi-node deployment for Integrator

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: integrator/bosh-release/config/final.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (VimukthiPerera) [inline integrator/bosh-release/config/final.yml]: Shall we put the latest copyright notice?

File: integrator/bosh-release/create.sh [hunk 2]
```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (VimukthiPerera) [inline integrator/bosh-release/create.sh]: The copyright notice should be updated.

File: integrator/bosh-release/export.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (VimukthiPerera) [inline integrator/bosh-release/export.sh]: The copyright notice should be updated here as well.

File: integrator/bosh-release/jobs/nfs_server/templates/ctl.erb [hunk 4]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (VimukthiPerera) [inline integrator/bosh-release/jobs/nfs_server/templates/ctl.erb]: Shall we update the copyright notice here as well?

File: integrator/bosh-release/jobs/wso2ei/spec [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+name: wso2ei
+templates:
+  ctl.erb: bin/ctl
+  conf/datasources/master-datasources.xml: conf/datasources/master-datasources.xml
```
Comment (VimukthiPerera) [inline integrator/bosh-release/jobs/wso2ei/spec]: The templated should be in the .erb format in order to apply the changes.

File: integrator/bosh-release/jobs/wso2ei/templates/conf/datasources/master-datasources.xml [hunk 6]
```diff
@@ -0,0 +1,68 @@
+<datasources-configuration xmlns:svns="http://org.wso2.securevault/configuration">
+  
+    <providers>
+        <provider>org.wso2.carbon.ndatasource.rdbms.RDBMSDataSourceReader</provider>
+    </providers>
+  
+    <datasources>
+      
+        <datasource>
+            <name>WSO2_CARBON_DB</name>
```
Comment (VimukthiPerera) [inline integrator/bosh-release/jobs/wso2ei/templates/conf/datasources/master-datasources.xml]: We need to have separate configurations for the registry, user-mgt etc. Shall we introduce those changes as well?

File: integrator/tile/build.sh [hunk 7]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# ------------------------------------------------------------------------
+#
+# Copyright 2017 WSO2, Inc. (http://wso2.com)
```
Comment (VimukthiPerera) [inline integrator/tile/build.sh]: Shall we update the copyright notice here as well?

File: integrator/tile/tile.yml [hunk 8]
```diff
@@ -0,0 +1,106 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2017 WSO2, Inc. (http://wso2.com)
```
Comment (VimukthiPerera) [inline integrator/tile/tile.yml]: The copyright notice should be updated.

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: integrator/bosh-release/jobs/wso2ei/templates/conf/datasources/master-datasources.xml (hunk 6 above)
Comment (Savidude) [reply integrator/bosh-release/jobs/wso2ei/templates/conf/datasources/master-datasources.xml]: Created an issue for this in https://github.com/wso2/pivotal-cf-ei/issues/2. Will fix with a pull request in future.

**Review by VimukthiPerera — APPROVED** [review]

File: integrator/bosh-release/jobs/wso2ei/templates/conf/datasources/master-datasources.xml (hunk 6 above)
Comment (VimukthiPerera) [reply integrator/bosh-release/jobs/wso2ei/templates/conf/datasources/master-datasources.xml]: +1

**Review by msmshariq — APPROVED** [review]

**Pull request merged**